engine/src/test/java/org/pentaho/di/trans/steps/tableoutput/TableOutputTest.java

```diff
@@ -202,4 +220,77 @@ public void testInit_unsupportedConnection() {
     KettleException ke = new KettleException( unsupportedTableOutputMessage );
     verify( tableOutputSpy, times( 1 ) ).logError( "An error occurred intialising this step: " + ke.getMessage() );
   }
+
+  @Test
+  public void validationRowMetaTest() throws Exception {
+    Method m = TableOutput.class.getDeclaredMethod( "isValidRowMeta", RowMetaInterface.class );
+    m.setAccessible( true );
+    Object result1 = m.invoke( null, filled );
+    Object result2 = m.invoke( null, empty );
+    assertTrue( Boolean.parseBoolean( result1 + "" ) );
```

Comment:
This is printing and parsing a boolean value, could just check the value directly.

Reply:
The test case is moved from dialog to particular step class.
Please refer master test class -  https://github.com/pentaho/pentaho-kettle/blob/master/ui/src/test/java/org/pentaho/di/ui/trans/steps/tableoutput/TableOutputDialogTest.java#L59

Reply:
If you're moving both the test and the target method, it would seem to me like the right time to clean it up, especially given how trivial the change is.